src/keyfile.c

```diff
@@ -73,37 +73,40 @@
 
 /* some default settings which are used at the very first start of Geany to fill
  * the configuration file */
-#define GEANY_MAX_SYMBOLLIST_HEIGHT		10
-#define GEANY_MIN_SYMBOLLIST_CHARS		4
-#define GEANY_MSGWIN_HEIGHT				208
-#define GEANY_DISK_CHECK_TIMEOUT		30
-#define GEANY_DEFAULT_TOOLS_MAKE		"make"
+#define GEANY_MAX_SYMBOLLIST_HEIGHT			10
+#define GEANY_MIN_SYMBOLLIST_CHARS			4
+#define GEANY_MSGWIN_HEIGHT					208
+#define GEANY_DISK_CHECK_TIMEOUT			30
+#define GEANY_DEFAULT_TOOLS_MAKE			"make"
```

Comment:
Why all these noise changes?

Reply:
These are to realign all the columns after adding `GEANY_DEFAULT_TOOLS_FILE_MANAGER` which is longer than the previous items.

Reply:
Just to mention, we prefer not to realign lines when we add fields. From HACKING:
```c
typedef struct Foo
    {
        gint    foo;    /* names are somewhat aligned visually */
        gint    bar;    /* fields don't share the same line */
        SomeLongTypeName baz; /* alignment is not strict */
```